targets/targets.json

```diff
@@ -5270,6 +5270,31 @@
             "target.network-default-interface-type": "ETHERNET"
         }
     },
+    "S5JS100": {
+        "inherits": ["Target"],
+        "core": "Cortex-M7",
+        "components_add": ["FLASHIAP"],
+        "supported_toolchains": ["GCC_ARM", "IAR", "ARM"],
+        "default_toolchain": "GCC_ARM",
+        "extra_labels": ["Samsung", "SIDK_S5JS100"],
+        "macros": ["CMSDK_CM7", "MBED_MPU_CUSTOM"],
+        "device_has": [
+            "ANALOGIN",
+            "I2C",
+            "INTERRUPTIN",
+            "PORTIN",
+            "PORTINOUT",
+            "PORTOUT",
+            "SERIAL",
+            "SPI",
+            "SLEEP",
+            "FLASH",
+            "STORAGE"
```

Comment:
Same here - should be removed.

Reply:
Thanks. I fixed it on e578320f5aea90759f813e9a6f1eea2d2f884936.